src/test/java/seedu/address/logic/commands/edit/EditCommandTest.java

```diff
@@ -206,4 +224,35 @@ public void toStringMethod() {
         assertEquals(expected, editCommand.toString());
     }
 
+    @Test
+    public void getChangeDescription_hasChanges_success() {
+        // changed all fields
+        Person person = new PersonBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
+                .withEmail(VALID_EMAIL_AMY).withEmptyAddress()
+                .withTags(VALID_TAG_HUSBAND).withModuleRoleMap(new ModuleCode("MA1522"), RoleType.TUTOR).build();
+
+        Person editedPerson = new PersonBuilder().withName(VALID_NAME_BETTY).withPhone(VALID_PHONE_BETTY)
+                .withEmail(VALID_EMAIL_BETTY).withAddress(VALID_ADDRESS_AMY)
+                .withTags(VALID_TAG_FRIEND).withModuleRoleMap(new ModuleCode("MA2001"), RoleType.PROFESSOR).build();
+
+        String expected = "Changes made: "
```

Comment:
If you decide to change this to `Change(s)` made, remember to update such test cases